src/utils/hooks.tsx

```diff
@@ -1,7 +1,16 @@
-import { useEffect } from 'react';
+import { useEffect, useState } from 'react';
 import { useNavigate, useParams } from 'react-router';
 import { get_test, TestData } from './testdata';
 
+export type JSONValue<T = any> =
```

Comment:
Is this really necessary? It just evaluates to `any` in every place it is used in this project, or am I missing something?

Reply:
ok, removed it